tests/bluetooth/bsim_bt/bsim_test_gatt_write/src/main.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+
+#include <zephyr/zephyr.h>
+
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys/util.h>
+
+#include "bs_types.h"
+#include "bs_tracing.h"
+#include "time_machine.h"
+#include "bstests.h"
+
+#define COUNT      5000   /* Arbitrary GATT Write Cmd iterations used */
+#define WRITE_RATE 780800 /* GATT Write bps recorded in this test */
```

Comment:
I assume this is a value measured at some point, and that it should also be able to be calculated as a function with connection interval and write data length provided?

Reply:
Added calculations in comment.